ftdc/cmd/parser.go

```diff
@@ -196,26 +204,28 @@ func main() {
 		}
 		render = true
 
+		// This is a CLI. It's acceptable to output to stdout.
+		//nolint:forbidigo
 		fmt.Print("$ ")
 		cmd, err := stdinReader.ReadString('\n')
 		switch {
 		case err != nil && errors.Is(err, io.EOF):
-			fmt.Println("\nExiting...")
+			nolintPrintln("\nExiting...")
 			return
 		case cmd == "quit":
-			fmt.Println("Exiting...")
+			nolintPrintln("Exiting...")
 			return
 		case cmd == "h" || cmd == "help":
 			render = false
-			fmt.Println("range <start> <end>")
-			fmt.Println("-  Only plot datapoints within the given range. \"zoom in\"")
-			fmt.Println("-  E.g: range 2024-09-24T18:00:00 2024-09-24T18:30:00")
-			fmt.Println("-  All times in UTC")
-			fmt.Println()
-			fmt.Println("reset range")
-			fmt.Println("-  Unset any prior range. \"zoom out to full\"")
-			fmt.Println()
-			fmt.Println("`quit` or Ctrl-d to exit")
+			nolintPrintln("range <start> <end>")
+			nolintPrintln("-  Only plot datapoints within the given range. \"zoom in\"")
+			nolintPrintln("-  E.g: range 2024-09-24T18:00:00 2024-09-24T18:30:00")
+			nolintPrintln("-  All times in UTC")
+			nolintPrintln()
+			nolintPrintln("reset range")
+			nolintPrintln("-  Unset any prior range. \"zoom out to full\"")
+			nolintPrintln()
+			nolintPrintln("`quit` or Ctrl-d to exit")
```

Comment:
Not `ctrl-c`?

Reply:
When the terminal gets `Ctrl-d`, it closes stdin. It's common for REPLs to specially handle `Ctrl-c`, see python:
```
dgottlieb@empanada:~/viam/rdk (ftdc-rotation) $ python
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>  <--- was Ctrl-c
KeyboardInterrupt
>>> <--- was Ctrl-d
dgottlieb@empanada:~/viam/rdk (ftdc-rotation) $
```

Reply:
That said -- being I don't catch SIGINT, Ctrl-c will also exit the program.

Reply:
Sounds good; didn't really realize this was a REPL until you demoed it the other day.